### PR TITLE
Fix minor bugs & improvements

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module gsclient-go
+
+go 1.13
+
+require (
+	github.com/google/uuid v1.1.1
+	github.com/gridscale/gsclient-go v2.1.0+incompatible
```
Comment (itakouna) [inline go.mod]: I think  `github.com/gridscale/gsclient-go v2.1.0+incompatible` should not be part of the requirements.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module gsclient-go
```
Comment (itakouna) [inline go.mod]: should not be the full path e.g, `github.com/gridscale/gsclient-go`?

File: go.mod (hunk 1 above)
Comment (nvthongswansea) [reply go.mod]: This is automatically added by gomod as we use it in examples.

File: go.mod (hunk 2 above)
Comment (nvthongswansea) [reply go.mod]: Again, go.mod file is automatically generated by command `go mod init`

Comment (itakouna) [reply go.mod]: Thanks @nvthongswansea for the clarification.

**Pull request merged**